rollup/base-config.js

```diff
@@ -14,8 +15,9 @@ const plugins = [
 
   replace({
     values: {
-      __DEV__: JSON.stringify(DEV),
-      __DEMO__: JSON.stringify(DEMO),
+      ___DEV___: JSON.stringify(DEV),
```

Comment:
So the warning module required by polyglot includes [this expression](https://github.com/BerkeleyTrue/warning/blob/master/warning.js#L19). If we replace `__DEV__` here it causes errors. The easy fix I put in here is just make our internal flag three underscores to avoid conflict. Another option I tried was just using rollup to eliminate that whole line, since __DEV__ would be replaced anyway. It's dirtier, but maybe an improvement. I'm fine with whichever you prefer.